pom.xml

```diff
@@ -613,6 +619,61 @@ under the License.
 
 	<profiles>
 
+		<profile>
+			<id>scala-2.12</id>
+			<activation>
+				<property>
+					<name>scala-212</name>
+				</property>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>build-helper-maven-plugin</artifactId>
+						<version>1.12</version>
+						<executions>
+							<execution>
+								<id>regex-property</id>
+								<goals>
+									<goal>regex-property</goal>
+								</goals>
+								<configuration>
+									<name>maven.test.skip</name>
+									<value>${project.artifactId}</value>
+									<regex>(flink-connector-kafka-0.9.*)|(flink-connector-kafka-0.8.*)|(flink-scala-shell.*)</regex>
```

Comment:
Kafka 0.9 and the scala shell theoretically work with Scala 2.12. For Kafka 0.9 we can't run tests because there is no `kafka_2.12` dependency for 0.9. Some Scala Shell tests pass and some don't but some stuff seems to work.

Reply:
Kafka 0.8 can be removed here because we actually exclude it from the build when building for Scala 2.12.

Reply:
What exactly does _theoretically_ mean here? Do i understand it correctly that the connector works but the test infrastructure doesn't? Are the scala-shell tests failing because the test infrastructure is broken, or because the shell is in a half-broken state now? (in the latter case I'd exclude it completely)

Reply:
I'll exclude the scala shell completely. And yes, for Kafka 0.9 the connector should work but testing doesn't work because the test dependency is not available for Scala 2.12